org.flatpak.Gourmet.yml

```diff
@@ -0,0 +1,54 @@
+app-id: org.flatpak.Gourmet
+runtime: org.gnome.Platform
+runtime-version: '3.36'
```

Comment:
Could you shed some light on this number?

Reply:
Flatpaks target a runtime, sdk, and version.  KDE, Gnome, and Freedesktop are the choices, along with their versions.  I went with Gnome 3.36 for the flatpak.